src/components/Block/index.tsx

```diff
@@ -11,6 +11,7 @@ interface BlockProps {
 }
 
 const Block: React.FC<BlockProps> = ({ panels }) => {
+  // @ts-ignore
```

Comment:
Would be handy to have a brief comment here about why the ignore is applied.

Reply:
Thanks. I thought a small refactor to isolate the ts-ignore and leave it typed in the rest of the app would be better.

It blew out into a huge refactor, but I guess that's how it always goes. Anyway might be worth another look.